frontend/src/components/BookTable/BookControl.tsx

```diff
@@ -359,7 +359,8 @@ const BookControl = ({
                     >
                       <div style={{ display: 'flex', alignItems: 'center', flexWrap: 'wrap' }}>
                         <RobotAvatar
-                          shortAlias={coordinator.shortAlias}
+                          shortAlias={coordinator.federated ? coordinator.shortAlias : undefined}
+                          hashId={coordinator.federated ? undefined : coordinator.shortAlias}
```

Comment:
The `coordinator.mainnet.onion` fix is needed here too so the avatar is the same as in the `BookTable`